src/components/AdvancedSwapDetailsToggle/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+import { ChevronDown, ChevronUp } from 'react-feather'
+import { Flex } from 'rebass'
+
+const StyledAdvancedSwapDetailsButton = styled(Flex)`
+  cursor: pointer;
+  background: rgba(104, 110, 148, 0.3);
+  border-radius: 4px;
+  height: 18px;
+  width: 22px;
+`
```

Comment:
Any reason why not extend `Button`?

Reply:
Don't like the predefined Button styles that we have to override but accessibility was lacking in this case. Will change to `Button`